srcpkgs/workrave/template

```diff
@@ -16,7 +16,7 @@ maintainer="Orphaned <orphan@voidlinux.org>"
 license="GPL-3.0-or-later"
 homepage="http://www.workrave.org/"
```

Comment:
https

Reply:
Sorry, have been away: completely missed that when I pushed it. Will fix now.